source/userguide/deep_neural_nets.rst

```diff
@@ -0,0 +1,1848 @@
+|                                         | Type: bool                                                                                                                                                                                                                                       |
+|                                         |                                                                                                                                                                                                                                                  |
+|                                         | Default: ``False``                                                                                                                                                                                                                               |
+|                                         |                                                                                                                                                                                                                                                  |
+|                                         | Choices: ``True``, ``False``                                                                                                                                                                                                                     |
++-----------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+Below is a minimalistic example of a token classifier configuration for classifier based on CNN-LSTM:
+
+.. code-block:: python
+
+   {
+    'model_type': 'tagger',
+    'train_label_set': 'train.*\.txt',
+    'test_label_set': 'test.*\.txt',
+    'model_settings': {'classifier_type': 'cnn-lstm'},
+    'params': {
+        'embedder_type': 'glove',
+        'lstm_hidden_dim': 128,
+        'lstm_bidirectional': True,
+        'char_emb_dim': 32
+    },
+   }
+
+4. ``'lstm-lstm'`` classifier type
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. _dnns_token_classification_models_lstm_lstm:
+
+:wiki_api:`Long short-term memory networks (LSTM) <Long_short-term_memory>` based text classifiers are classifiers that utilize recurrent feedback connections to be able to learn temporal dependencies in sequential data.
+When coupled with :wiki_api:`Long short-term memory networks (LSTM) <Long_short-term_memory>` for extracting character-level features from input text, the overall architecture can better model the textual data as well as it is more robust to variations in the spellings.
+In its typical architecture for entity recognition, the first layer embeds the sequence of words obtained f
```

Comment:
This section is structured differently than the other 4 sections of this type, in that it has a straightforward bulleted list of the key points for this section. The other sections do a good job of laying out the key points, too, but they do so in paragraph form. 

I'm kind of partial to this display of information, though it is a bit of a departure from what appears to be the norm for MindMeld documentation. On the one hand, it might be a lot of work to rewrite the other sections in this 'Entity recognition'  section as opposed to making this conform to the style imposed in those other sections. But on the other hand, this is a really nice point-for-point outline of what is happening with this classifier type, and it's presented in a very user-friendly visual and organizational style.